src/common/ceph_crypto.cc

```diff
@@ -27,7 +27,7 @@ void ceph::crypto::init(CephContext *cct)
 {
 }
 
-void ceph::crypto::shutdown(bool shared)
+void ceph::crypto::shutdown(bool)
```

Comment:
Yes, CPython enforces this since the core dev will do the squash when merging.
http://cpython-devguide.readthedocs.io/pullrequest.html#submitting